### ISSUE TITLE
Forking dnslib

### ISSUE THREAD

**Issue by nhairs** [Alviner/dnslib3#13]
Hi @mosquito, @Alviner, 

I've noticed a few pull requests against [paulc/dnslib](https://github.com/paulc/dnslib) which appear to be originating from this repository.

Before continuing, you might want to take a look at my PR https://github.com/paulc/dnslib/pull/61 where I did a lot of work to update dnslib to python 3.

In addition it looks like the original author (paulc) would prefer to have dnslib forked rather than continue under it's current name: https://github.com/paulc/dnslib/pull/61#issuecomment-2003796960

Depending on your intentions it might be worth starting a discussion with paulc if you did want to continue to maintain an updated dns lib into the future. In this regard I would be happy to contribute the code in that PR to teh cause, and may be able to help maintain it (though do not have the time to do heaps of feature work at the moment).

That said [per my comment on that PR](https://github.com/paulc/dnslib/pull/61#issuecomment-1920831971), unless you /really/ want to use dnslib, it might be better to focus efforts on the much more active dnspython rather than having 2 communities attempting the same thing.

**Comment by mosquito** [comment]
Yep, I saw it. My aim the change this library and do not afraid the breaking changes. I want to rewrite some things using dataclasses and enum modules.

**Comment by Alviner** [comment]
@nhairs Hi,

Thank you for reaching out and for your work on the dnslib project.

We have indeed decided to fork the package and will continue under the name [dnslib3](https://pypi.org/project/dnslib3/). We appreciate your efforts on the pull request; however, it changes too much at once and loses a lot of the history.

That said, PRs are always welcome! We have already integrated pyproject, pytest and are in the process of refactoring, during which we will also incorporate typing with a linter.

We would be happy to have you contribute to the new repository if you are interested.
